pkg/manifest/helm_processor.go

```diff
@@ -110,22 +111,30 @@ func (h *helm) Install(stringifedManifest string, deployInfo types.InstallInfo,
 		return false, err
 	}
 
+	for i := range postRuns {
```

Comment:
should we put the `postRuns` at the end of this function?

Reply:
I think you maybe want to review the other PR since this one is mainly targeting the new declarative lib. WDYT?